GuestDetails/Guest-Details.html

```diff
@@ -0,0 +1,83 @@
+	<!doctype>
+	<!DOCTYPE html>
+	<html>
+	<head>
+		<title>GuestDetails</title>
+		<link rel="stylesheet" type="text/css" href="Guest-Details-style.css">
+		<script src="Guest-Details.js"></script>
+	</head>
+	<body>
+
+		<div class="main-container">
+			<div id="header">
+						<div id="logo"> 
+							 <img src="download.png" alt="Expedia">
+						</div>
+				<div id="wrapper">
+					<h1>Review And Book</h1>
+					<h3>Act fast!! Pricing and availability may change</h3>
+					<h4>Great choice! You choose the cheapest room</h4>
+
+					<p>Room 1</p>
+					<div id="parent-container">
+							<div class="container">
+										    <label><b>Contact Name</b></label>
+										    <input type="text" placeholder="Contact Name" name="contact Name" required>
+										    <br>
+										    <label><b>Country Code</b></label>
+										    <input type="Country Code" placeholder="Country Code" name="code" required>
```

Comment:
Use a drop down list containing the name and country code for different countries.